packages/block-editor/src/components/floating-toolbar/index.native.js

```diff
@@ -18,30 +25,88 @@ import styles from './styles.scss';
 import NavigateUpSVG from './nav-up-icon';
 import Breadcrumb from '../block-list/breadcrumb.native';
 
+const EASE_IN_DURATION = 250;
+const EASE_OUT_DURATION = 80;
+const TRANSLATION_RANGE = 8;
+
+const opacity = new Animated.Value( 0 );
+
 const FloatingToolbar = ( {
 	selectedClientId,
 	parentId,
 	showFloatingToolbar,
 	onNavigateUp,
 	isRTL,
-} ) =>
-	!! showFloatingToolbar && (
-		<TouchableWithoutFeedback accessible={ false }>
-			<View style={ styles.floatingToolbar }>
-				{ !! parentId && (
-					<Toolbar passedStyle={ styles.toolbar }>
-						<ToolbarButton
-							title={ __( 'Navigate Up' ) }
-							onClick={ () => onNavigateUp( parentId ) }
-							icon={ <NavigateUpSVG isRTL={ isRTL } /> }
-						/>
-						<View style={ styles.pipe } />
-					</Toolbar>
-				) }
-				<Breadcrumb clientId={ selectedClientId } />
-			</View>
-		</TouchableWithoutFeedback>
+} ) => {
+	// sustain old selection for proper Breacdrumb rendering when exit animation is ongoing
+	const [ previousSelection, setPreviousSelection ] = useState( {} );
+
+	useEffect( () => {
+		const easing = Easing.ease;
+		Animated.timing( opacity, {
+			toValue: showFloatingToolbar ? 1 : 0,
+			duration: showFloatingToolbar
+				? EASE_IN_DURATION
+				: EASE_OUT_DURATION,
+			easing,
```

Comment:
I see no point in creating the easing const to use it here.

```suggestion
			Easing.ease,
```

Reply:
done